src/frontend/komponenter/Fagsak/journalposter/JournalpostListe.tsx

```diff
@@ -41,14 +50,47 @@ const StyledSidetittel = styled(Sidetittel)`
     margin-bottom: 1rem;
 `;
 
+const StyledTd = styled.td`
+    vertical-align: top;
+`;
+
+const StyledVedleggsliste = styled.ul`
```

Comment:
```suggestion
const Vedleggsliste = styled.ul`
```
Kan godt droppe Styled om det bare er styling på standard html elementer :)